testing/test_access

```diff
@@ -12,9 +12,16 @@ function send_udp {
     container=$1
     addr=$2
     count=$3
+    interval=
+    if [ -z "$4" ]; then
```

Comment:
This can be   interval=${4:-1}   -- which is bash syntax for default

Reply:
Done